task_2/src/server.ts

```diff
@@ -18,14 +19,27 @@ app.use('/user/', userRouter)
 app.use('/group/', groupRouter)
 app.use('/userGroup/', userGroupRouter)
 
+// middleware to log errors
+app.use(logUnhandledError)
```

Comment:
I think you got confused here... because lines from 37 to 45 do whatever you trying to do. why do you need it here?